desktop/apps/auction2/reducers.js

```diff
@@ -1,21 +1,123 @@
 import { combineReducers } from 'redux'
 import u from 'updeep'
+import { contains } from 'underscore'
+import { data as sd } from 'sharify'
+import * as actions from './actions'
 
 const initialState = {
+  minEstimateRangeDisplay: 50,
+  maxEstimateRangeDisplay: 50000,
   artworks: [],
-  isListView: false
+  total: 0,
+  isListView: false,
+  isFetchingArtworks: false,
+  sortMap: {
+    "lot_number": "Lot Number (asc.)",
+    "-lot_number": "Lot Number (desc.)",
+    "-searchable_estimate": "Most Expensive",
+    "searchable_estimate": "Least Expensive"
+  },
+  aggregatedArtists: [],
+  aggregatedMediums: [],
+  filterParams: {
+    size: 50,
+    page: 1,
+    sale_id: sd.AUCTION.id,
+    aggregations: ['TOTAL', 'MEDIUM', 'FOLLOWED_ARTISTS', 'ARTIST'],
+    sort: 'lot_number',
+    gene_ids: [],
+    artist_ids: [],
+    estimate_range: '',
+    ranges: {
+      estimate_range: {
+        min: 50,
+        max: 50000
+      }
+    }
+  }
```

Comment:
This is something that I'm always kinda dropping here and there as a hint, but I can't emphasize how good of a habit is is that will carry you far... (and strangely, people don't _really_ realize it until they've realized it some way or another, its non-obvious)... always alphabetize! Alphabetize _everything_ -- imports, variables, object properties, etc. Built into your editor is a `sort` command, and if you highlight the lines and do `sort: natural` then it will do it automatically, including `CapsCase` and `*` (various symbols). When another developer comes along and sees code that they didn't write, having things sorted by alpha makes makes things much more easy to read (the eyes naturally flow from a-z) as well as maintain when going back later and updating. [Reducer props](https://github.com/artsy/prediction/blob/master/src/reducers/index.js#L27) are a really good example of this, and [so are import statements](https://github.com/artsy/prediction/blob/master/src/server.js#L1) but [also metaphysics queries](https://github.com/artsy/metaphysics/pull/561) :) In the end this is just a matter of opinion, but I definitely regard it as a secret weapon that people don't use enough (or really "know" about). It takes a lot of thinking right out of things, especially over time.